vcd/resource_vcd_vapp.go

```diff
@@ -151,6 +152,12 @@ func resourceVcdVAppUpdate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error finding VApp: %#v", err)
 	}
 
+	if d.HasChange("name") || d.HasChange("description") {
```

Comment:
I guess this can be removed just to avoid confusion:
```suggestion
	if d.HasChange("description") {
```

Reply:
changed